components/link/link-mixin.js

```diff
@@ -0,0 +1,81 @@
+import '../icons/icon.js';
+import { css, html, nothing } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { LocalizeCoreElement } from '../../helpers/localize-core-element.js';
+import { offscreenStyles } from '../offscreen/offscreen.js';
+
+/**
+ * A menu item component used for navigating.
+ * @fires click - Dispatched when the link is clicked
+ * @slot supporting - Allows supporting information to be displayed on the right-most side of the menu item
+ */
+export const LinkMixin = superclass => class extends LocalizeCoreElement(superclass) {
+
+	static get properties() {
+		return {
+			/**
+			 * Prompts the user to save the linked URL instead of navigating to it.
+			 * Must be to a resource on the same origin.
+			 * Can be used with or without a value, when set the value becomes the filename.
+			 * @type {string}
+			 */
+			download: { type: String },
```

Comment:
My only worry here is that by (correctly) treating it as a string, we might somehow break a consumer that was passing it as a boolean. The only use I could find [was this one](https://github.com/Brightspace/completions-ui/blob/b3d453715b24824c40d8fbef6f56441d6d842f71/src/course-completion-migration.js#L227), which I think is still going to work?